### PR TITLE
MBR analysis

### PR DESCRIPTION
**Pull request by wtdcode**
Fully emulated Petya and tons of implementation for DOS/BIOS interrupts.

### CODE REVIEW

File: qiling/os/disk.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    @property
+    def host_path(self):
+        return self._host_path
+    
+    @property
+    def drive_path(self):
+        return self._drive_path
+    
+    @property
+    def f(self):
```
Comment (xwings) [inline qiling/os/disk.py:63]: fs ? something more clearer then f

**Comment by aquynh** [comment]
please have docs, too.

**Comment by wtdcode** [comment]
https://github.com/qilingframework/rtfd.io/pull/9

Docs PR.

**Comment by xwings** [comment]
cool.

**Pull request merged**